components/core/tools/docker-images/clp-env-base-musllinux_1_2/Dockerfile

```
@@ -1,6 +1,8 @@
 # syntax=docker/dockerfile:1
 
-FROM quay.io/pypa/musllinux_1_2_aarch64
+# Multi-arch base image (amd64, arm64, ...). Build with `--platform` to select the
+# target architecture; see build.sh (PLATFORM env var for local cross-builds).
+FROM quay.io/pypa/musllinux_1_2
```

Hadolint (2.14.0)

[warning] Always tag the version of an image explicitly

(DL3006)